entrypoint.sh

```diff
@@ -17,11 +17,15 @@ create_bind_data_dir() {
   rm -rf /etc/bind
   ln -sf ${BIND_DATA_DIR}/etc /etc/bind
   chmod -R 0775 ${BIND_DATA_DIR}
-  chown -R ${BIND_USER}:${BIND_USER} ${BIND_DATA_DIR}
```

Comment:
I think a simpler and less verbose way to achieve this feature is with a `chown` function, fe.

```bash
chown() {
  [[ $SKIP_CHOWN != "true" ]] && /bin/chown $@
}
```